packages/ilios-common/addon/components/course/objective-list.js

```diff
@@ -67,10 +79,9 @@ export default class CourseObjectiveListComponent extends Component {
     return this.args.course.hasMany('courseObjectives').ids().length;
   }
 
-  load = restartableTask(async () => {
-    //pre-load all session data as well to get access to child objectives
-    await this.dataLoader.loadCourseSessions(this.args.course.id);
-  });
+  async getCourseSessions(course) {
+    await this.dataLoader.loadCourseSessions(course.id);
+  }
```

Comment:
i think all of this can be scrapped without replacement. will send a follow-up PR for this.

Reply:
I ventured a guess that this could be done further upstream somehow (in the course route?), but wasn't sure how.